views/Main.react.js

```diff
@@ -501,16 +479,17 @@ class MainPage extends Component {
       body: JSON.stringify({
         preferences: preference
       })
-    }).then((response) => {
+    }).then(() => {
```

Comment:
Even though `response` variable is not used downstream, was it necessary to remove that?

Reply:
Not strictly necessary but it was flagged by ESLint, and I wanted to eliminate all ESLint warnings possible. It is a cleanup task.